app/src/main/java/com/example/benchproject/data/movies/common/constants/MoviesConstants.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.benchproject.data.movies.common.constants
+
+class MoviesConstants {
```

Comment:
I think that you added it in other PR? Some rebase problems?

Reply:
The same problem as above. This is not in this PR, when I added new changes from the previous PR these changes disappeared